diff --git a/dist/transitions/router.js b/dist/transitions/router.js
index c8e68d7e65b5a8bf7a44c9e27a9793b170fdaebe..74192406fa6f727f2b0e6daf34ce827955fb4434 100644
--- a/dist/transitions/router.js
+++ b/dist/transitions/router.js
@@ -42,10 +42,10 @@ let currentHistoryIndex = 0;
 if (inBrowser) {
   if (history.state) {
     currentHistoryIndex = history.state.index;
-    scrollTo({ left: history.state.scrollX, top: history.state.scrollY });
+    history.scrollRestoration = "auto";
   } else if (transitionEnabledOnThisPage()) {
     history.replaceState({ index: currentHistoryIndex, scrollX, scrollY }, "");
-    history.scrollRestoration = "manual";
+    history.scrollRestoration = "auto";
   }
 }
 async function fetchHTML(href, init) {
